src/explore-education-statistics-frontend/src/middleware/pages/redirectPages.ts

```diff
@@ -0,0 +1,62 @@
+import redirectService, {
+  Redirects,
+  RedirectType,
+} from '@frontend/services/redirectService';
+import type { NextRequest } from 'next/server';
+import { NextResponse } from 'next/server';
+
+interface StoredRedirects {
+  redirects: Redirects;
+  fetchedAt: number;
+}
+
+let storedRedirects: StoredRedirects | undefined;
+
+const redirectPaths = {
+  methodologyRedirects: '/methodology',
+  publicationRedirects: '/find-statistics',
+};
+
+const cacheTime = 300000;
```

Comment:
Is this 5 mins?  Might be worth just adding a comment to that effect.  Might potentially be a little high as well - but Cam would probably be best placed to make the call on that, if this didn't come from him.

Reply:
We can also format the number a little nicer too i.e. `300_000`